src/librustc_passes/layout_test.rs

```diff
@@ -0,0 +1,132 @@
+use rustc::hir;
```

Comment:
Interesting approach. I would’ve expected that a few codegen tests would’ve sufficed, but I guess this still tests more thoroughly than a codegen test of a similar size.